api/app/Models/Department.php

```diff
@@ -53,12 +53,6 @@ public function poolCandidateSearchRequests(): HasMany
         return $this->hasMany(PoolCandidateSearchRequest::class);
     }
 
-    /** @return BelongsToMany<Team, $this> */
-    public function teams(): BelongsToMany
-    {
-        return $this->belongsToMany(Team::class, 'team_department');
-    }
-
```

Comment:
Remember to wipe it from the Schema for both relations

![image](https://github.com/user-attachments/assets/b63dcb1a-85fb-43b0-a6c6-98f01cdcfb64)

Reply:
02ff17e3f7e5d6231efd31c43751b826e851bfa3